fiftyone/zoo/models/manifest-torch.json

```diff
@@ -1673,6 +1673,53 @@
                 "torch"
             ],
             "date_added": "2020-12-11 13:45:51"
+        },
+        {
+            "base_name": "clip-vit-base32-torch",
```

Comment:
Nice work figuring out how to use this! 👏

Reply:
took a bit of time, but now should be easier